chapter1.md

```diff
@@ -43,55 +37,63 @@ All abundances in the compiled datasets were counts of individuals.
+The majority of species abundance distributions (SADs) are constructed using counts of individuals [for discussion of alternative approaches see @mcgill2007 and @morlon2009]. As such, the data are discrete and discrete distributions (i.e., those that only have non-zero probabilities for positive integer values of abundance) are more appropriate.
 
-McGill et al. [-@mcgill2007] classified species abundance distribution models into five different families: purely statistical, branching process, population dynamics, niche partitioning, and spatial distribution of individuals. We evaluated models from each of the separate families, excluding the spatial distribution family, which requires spatially explicit data. Specifically, we evaluated the log-series, the Poisson lognormal, the negative binomial, and the Zipf distributions. All distributions were defined to be capable of having non-zero probability at values from 1 to infinity. We excluded models from analysis that do not have explicit likelihoods [e.g., geometric; @motomura1932, some niche partitioning models; @sugihara1980; @tokeshi1993, double geometric; @alroy2015] so we could use likelihood based methods for fitting and evaluating distributions (see Analysis).
+McGill et al. [-@mcgill2007] classified species abundance distribution models into five different families: purely statistical, branching process, population dynamics, niche partitioning, and spatial distribution of individuals. We evaluated models from each of these families, except for niche partitioning. We excluded niche partitioning models from the analysis (including both classic models like the geometric series and niche fraction models [@motomura1932; @sugihara1980; @tokeshi1993] and Alroy's [@-alroy2015] new double geometric model) because these models do not have explicit likelihoods and therefore would have prevented the use of likelihood based methods for fitting and evaluating the distributions (see Analysis). We evaluated the log-series, the Poisson lognormal, the negative binomial, and the Zipf distributions. All distributions were defined to be capable of having non-zero probability at integer values from 1 to infinity.
 
 The log-series is one of the first distributions used to describe the SAD, being derived as a purely statistical distribution by Fisher [-@fisher1943]. It has since been derived as the result of ecological processes, the metacommunity SAD for ecological neutral theory [@hubbell2001; @volkov2003], and several different maximum entropy models [@pueyo2007; @harte2008].
 
 The lognormal is one of the most commonly used distributions for describing the SAD [@mcgill2003] and has been derived as a null form of the distribution resulting from the central limit theorem [@may1975], population dynamics [@engen1996], and niche partitioning [@sugihara1980]. We use the Poisson lognormal because it is a discrete form of the distribution appropriate for fitting discrete abundance data [@bulmer1974].
```

Comment:
If lognormal has been derived from niche partitioning, this would mean that we have covered all categories in McGill et al. 2007?